conbench/entities/benchmark_result.py

```diff
@@ -99,9 +100,17 @@ class BenchmarkResult(Base, EntityMixin):
     # store is the user-given timestamp properly translated to UTC.
     timestamp: Mapped[datetime] = NotNull(s.DateTime(timezone=False))
     iterations: Mapped[Optional[int]] = Nullable(s.Integer)
+
+    # Mean can only be None for errored BenchmarkResults. Is guaranteed to have
+    # a numeric value for all non-errored BenchmarkResults, including those
+    # that have just one data point (when the mean is not an exciting
+    # statistic, but still a useful one).
+    mean: Mapped[Optional[float]] = Nullable(s.Numeric, check("mean>=0"))
```

Comment:
Oh, I just realize. These are all type `Decimal` because of the default behavior of `s.Numeric`.